Portfolio/index.html

```diff
@@ -1,13 +1,34 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Portfolio</title>
-    <meta charset="UTF-8" />
-    <link rel="stylesheet" href="index.css" />
+    <title>Resume</title>
   </head>
 
-  <body style="margin: 0; padding: 0">
-    <!-- Write your code below for PORTFOLIO -->
-    <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
-  </body>
+  <body>
+    <h1>Jenny Doe</h1>
+    <img width = 100 src="https://www.pngjoy.com/pngm/258/4974996_welcome-girl-professional-young-woman-png-hd-png.png" alt="jenny">
+  <h1> Front End <br>Developer</h1>
+  <h4>GET TO KNOW ME</h4>
```

Comment:
The 'for' attribute in line 18 should have the same value as the 'id' attribute in line 17.